### ISSUE TITLE
[accessibility] Remove aria labels from non interactive blocks like spacer block.

### ISSUE THREAD

**Issue by dennisheiden** [WordPress/gutenberg#43378]
Please have a look into the specification here:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

"_The aria-label attribute defines a string value that labels an interactive element._"
and
"_Not all elements can be given an accessible name. Neither aria-label nor aria-labelledby should be used with non-interactive elements or inline structural role such as with code, term, or emphasis nor roles whose semantics will not be mapped to the accessibility API, including presentation, none, and hidden. **The aria-label attribute is intended for interactive elements only.** Use aria-label to ensure an accessible name is provided when none is visible in the DOM for all interactive elements, like links, videos, form controls, [landmark roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#3._landmark_roles), and [widget roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#2._widget_roles)._"

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

Spacer Block Markup:
![image](https://user-images.githubusercontent.com/25924499/185388383-52b2897a-b280-415b-9adc-4f784b973a4f.png)

Frontend output:
![image](https://user-images.githubusercontent.com/25924499/185393922-2cc82e77-d9b7-4f9c-8031-1a62e416035c.png)

## What is your proposed solution?
Remove aria-labels from non-interactive blocks. 

## Environments:
a) WordPress 6.0.1
b) WordPress 6.0.1 + Gutenberg 13.9

**Comment by skorasaurus** [comment]
Hi, 

Thanks for reporting and your concern about accessibility. 

The spacer block uses the [aria-hidden attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden), not aria-label. 

That said, the spacer block probably does not need the aria-hidden label, it is superfluous and there is an issue to track that at https://github.com/WordPress/gutenberg/issues/30157#issuecomment-1024436271

**Comment by dennisheiden** [comment]
Thank you for your clarification, looks like I was mixing up things but I'm aware of that now.
